src/GeniusYield/TxBuilder/NodeQuery.hs

```diff
@@ -129,7 +130,7 @@ instance GYTxQueryMonad GYTxQueryMonadNode where
         gyGetSlotOfCurrentBlock providers
 
     logMsg ns s msg = GYTxQueryMonadNode $ \(GYTxNodeEnv _ providers) ->
-        gyLog providers ns s msg
+        withFrozenCallStack $ gyLog providers ns s msg
```

Comment:
`withFrozenCallStack` is needed to correctly report for line location. This bug has been prevailing before the changes made to logging framework by this PR.